### PR TITLE
add license support

### PR DESCRIPTION
**Pull request by pdabelf5**
Add function GetNginxLicense() which returns a struct of type `NginxLicense`

### Proposed changes

Describe the use case and detail of the change. If this PR addresses an issue on GitHub, make sure to include a link to
that issue here in this description (not in the title of the PR).

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [ ] I have read the [CONTRIBUTING](https://github.com/nginxinc/nginx-plus-go-client/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all unit tests pass after adding my changes
- [ ] I have updated necessary documentation
- [ ] I have rebased my branch onto main
- [ ] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

File: client/nginx.go [hunk 1]
```diff
@@ -193,6 +193,20 @@ type NginxInfo struct {
...
+type LicenseReporting struct {
+	Healthy bool
+	Fails   uint64
+	Grace   uint64
+}
+
+// NginxLicense contains licensing information about NGINX Plus.
+type NginxLicense struct {
+	ActiveTill uint64 `json:"active_till"`
+	Eval       bool
```
Comment (j1m-ryan) [inline client/nginx.go:206]: Do we need json comments on these also, and the struct above, to tell it its lowercase?

Comment (pdabelf5) [reply client/nginx.go:206]: No, you only need the json hints if the parameters don't match, the capital in the struct doesn't matter

**Review by j1m-ryan — APPROVED** [review]

**Review by jjngx — APPROVED** [review]

**Review by AlexFenlon — APPROVED** [review]

**Review by AlexFenlon — APPROVED** [review]

File: client/nginx.go [hunk 2]
```diff
@@ -193,6 +196,20 @@ type NginxInfo struct {
...
+// LicenseReporting contains information about license status for NGINX Plus.
+type LicenseReporting struct {
+	Healthy bool
+	Fails   uint64
+	Grace   uint64
+}
+
+// NginxLicense contains licensing information about NGINX Plus.
+type NginxLicense struct {
+	ActiveTill uint64 `json:"active_till"`
```
Comment (sjberman) [inline client/nginx.go:208]: What is "till"?

Comment (pdabelf5) [reply client/nginx.go:208]: ActiveTill is a unix epoch

Comment (pdabelf5) [reply client/nginx.go:208]: It represents when the license expires.  `active_till` is what it is called in the nginx plus api.

**Review by j1m-ryan — APPROVED** [review]

**Review by sjberman — APPROVED** [review]

**Pull request merged**